src/main/java/nitezh/ministock/activities/GlobalWidgetData.java

```diff
@@ -1,20 +1,37 @@
 package nitezh.ministock.activities;
 
 import android.app.Application;
+import android.os.AsyncTask;
+import android.util.Log;
 
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import nitezh.ministock.activities.widget.WidgetRow;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
 import nitezh.ministock.activities.widget.WidgetRow;
-
+import java.util.Map;
+import java.util.Set;
```

Comment:
`import nitezh.ministock.activities.widget.WidgetRow;
import java.util.Map;
import java.util.Set;`

are not used and should be removed

Reply:
fixed